<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_SWNT</class>
 <widget class="QWidget" name="Form_SWNT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>311</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>273</width>
    <height>311</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>273</width>
    <height>311</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 223, 223);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <item row="0" column="0">
    <widget class="QWidget" name="SWNT" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(223, 223, 223);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="5" column="0">
           <widget class="QLabel" name="labl_properties_SWNT">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Nanotube properties</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_chirality_N_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>N:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox_chirality_N_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_chirality_M_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>M:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="spinBox_chirality_M_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_type_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Type:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_type1_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <item>
               <property name="text">
                <string>C</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>N</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>P</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="comboBox_type2_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <item>
               <property name="text">
                <string>C</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>B</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Al</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="lable_H_termination_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Hydrogen termination:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_H_termination_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>One end</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Both ends</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_bond_length_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Bond length (Å):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_bond_length_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="Diameter_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Diameter (Å):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_diameter_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_length_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Length (Å)  :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_length_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_repeat_units_SWNT">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Repeat units:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox_repeat_units_SWNT">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_help_build_SWNT">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_build_SWNT">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Build</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
